### ISSUE TITLE
Navbar Accessibility: Unable to use aria-label or alt to assign alternative text for screen readers

### ISSUE THREAD

**Issue by umang-sharma-ifx** [Infineon/infineon-design-system-stencil#2087]
### Detailed description of how to reproduce the problem

Unable to assign adequate alternate text to the Infineon Logo button in the Navbar as there is no aria-label exposed inside the ifx-navbar component. This effects screen readers who are unable to identify what the purpose of the button is, failing to meet accessibility criteria as outlined in _Additional Information_.

### Upload a screenshot

<img width="1023" height="693" alt="Image" src="https://github.com/user-attachments/assets/00901628-f56b-444d-952b-bf69ee9dfc48" />

<img width="455" height="213" alt="Image" src="https://github.com/user-attachments/assets/d9cf806b-bf50-4c58-9b1f-3880b2f82c98" />

### Describe the development framework

Vue.JS

### Expected behavior

Using aria-label in the code should display alternate text when the mouse hovers over the logo button that is also screen-reader accessible.

### Additional information

Accessibility Criteria outlined [here](https://www.w3.org/WAI/WCAG21/Understanding/).
Criteria **2.4.4**, **2.4.9**, and **4.1.2** are affected by this.
